config.yml

```diff
@@ -44,6 +44,11 @@ genesis:
     gov:
       deposit_params:
         min_deposit: [{ amount: "1", denom: "usge" }]
+    bet:
+      params:
+        placement_constraints:
+          min_amount: "1000000"
```

Comment:
we should follow consistency.
Either we add bet in every param like bet_fee and min_bet_amount
OR
fee and min_amount